docs/docs/FieldSet.mdx

```diff
@@ -0,0 +1,26 @@
+import { Story } from "../src/components/Story";
+
+# FieldSet
+
+> The accordion component allows the user to show and hide sections of related content on a page.
+>
+> \- [MUI Docs](https://mui.com/material-ui/react-accordion/)
+
+The Comet Admin `FieldSet` extends [Material UI's `Accordion`](https://mui.com/material-ui/react-accordion) by adding custom style and elements such as supportText, endAdornment, and options like collapsible and initialCollapsed.
+
+## Props
+
+| Name              | Type   | Default   | Description                                                                      |
+| :---------------- | :----- | :----- | :------------------------------------------------------------------------------- |
+| title          | ReactNode |  | Title of the FieldSet |
+| supportText          | ReactNode (optional) |  | Additional line below the Title |
+| endAdornment          | ReactNode (optional) |  | An element on the right end of the FieldSetSummary |
+| collapsible          | boolean (optional) | true | If true, FieldSet can be opened and closed (Accordion behaviour) |
+| initialCollapsed          | boolean (optional) | false | If true, FieldSet is closed by default |
+| disablePadding          | boolean (optional) | false | If true, content will have no padding |
+
+In addition to its own props, the FieldSet component also accepts all props from the Material UI Accordion component.
```

Comment:
Actually, it doesn't: The rest props are not passed down to the MUI Accordion component. And I'm not sure if they even should.